modules/mysql/read_replica.tf

```diff
@@ -37,6 +37,7 @@ resource "google_sql_database_instance" "replicas" {
 
   settings {
     tier                        = lookup(each.value, "tier", var.tier)
+    edition                     = lookup(each.value, "edition", var.edition)
```

Comment:
is `data_cache_config` applicable to mysql read replica also?

Reply:
Yes, verified it with mysql team and manually via creating replica with data cache on.

Reply:
so, we are adding the data_cache_config here also right?

Reply:
yes